package.json

```diff
@@ -41,6 +41,7 @@
   "homepage": "https://github.com/ChainSafe/js-libp2p-gossipsub#readme",
   "dependencies": {
     "@types/debug": "^4.1.7",
+    "@chainsafe/as-sha256": "^0.2.4",
```

Comment:
Maybe the fastMsgId should be passed into the constructor, like the msgId in the pubsub base class, so we don't need to add this dependency (maybe just dev dependency for testing).